L/libcvrpsep/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcvrpsep"
```

Comment:
Name of the package should match name of the upstream project, not the name of the product: https://docs.binarybuilder.org/stable/building/#Name
```suggestion
name = "CVRPSEP"
```
Note that you _**must**_ rename the directory accordingly (also move it under `C/` instead of `L/`)

Reply:
Thanks! Done.